src/lib/profiles/service-provisioning/ServiceProvisioning.h

```diff
@@ -161,7 +162,17 @@ class NL_DLL_EXPORT PairDeviceToAccountMessage
     static WEAVE_ERROR Decode(PacketBuffer *msgBuf, PairDeviceToAccountMessage& msg);
 };
 
+class NL_DLL_EXPORT IFJServiceFabricJoinMessage
+{
+public:
+    uint64_t ServiceId;
+    uint64_t FabricId;
+    const uint8_t *DeviceInitData;
+    uint16_t DeviceInitDataLen;
```

Comment:
I can't recall if we discuss this.  What is the purpose of the DeviceInitData in the context of IFJ?

Reply:
I think I recall we decided to leave the DeviceInitData(which essentially encodes the devicedescriptor) as that can be generated at the device.